app/resources/Base.lproj/BinaryTemplateController.xib

```diff
@@ -80,10 +80,10 @@
                     </tableHeaderView>
                 </scrollView>
                 <textField hidden="YES" horizontalHuggingPriority="251" verticalHuggingPriority="750" horizontalCompressionResistancePriority="250" fixedFrame="YES" setsMaxLayoutWidthAtFirstLayout="YES" translatesAutoresizingMaskIntoConstraints="NO" id="f71-AR-Dtg">
-                    <rect key="frame" x="-2" y="0.0" width="302" height="130"/>
+                    <rect key="frame" x="4" y="0.0" width="287" height="130"/>
                     <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
-                    <textFieldCell key="cell" controlSize="small" sendsActionOnEndEditing="YES" alignment="center" title="Label" id="3tC-5d-kwc">
-                        <font key="font" metaFont="controlContent" size="11"/>
+                    <textFieldCell key="cell" controlSize="small" sendsActionOnEndEditing="YES" alignment="left" title="Label" id="3tC-5d-kwc">
+                        <font key="font" size="12" name="Menlo-Regular"/>
```

Comment:
is there a more generic way to use a fixed width font? maybe we can do it in code instead of hard coding a font name.

Reply:
bump on this :)

Reply:
I believe this is the necessary change.
```suggestion
                        <font key="font" metaFont="fixedUser" size="11"/>
```